### PR TITLE
Lab3 Updates

### PR DESCRIPTION
**Pull request by Breanna-Geller**
Submission, Demo Instructions, Extra Help.
Thank you for your feedback.

Breanna

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,17 +59,48 @@ So, in summary, here's what we need to do:
...
+Implement wall following to make the car drive autonomously around the Levine Hall map. Follow the inner walls of Levine. Which means follow left- make sure the the car is going counter-clockwise in the loop. (The first race we run will be counter-clockwise). You can implement this node in either C++ or Python.
 
 ## V. Deliverables and Submission
 
-**Deliverable 1**: After you're finished, update the entire skeleton package directory with your `wall_follow` package and directly commit and push to the repo Github classroom created for you. Your commited code should start and run in simulation smoothly.
+**Deliverable 1**: After you're finished, update the entire skeleton package directory with your team's `wall_follow` package and directly commit and publish to GitHub. You will need to make sure the TA is added as a collaborator. Your committed code should start and run in simulation smoothly.The package should include a launch file that allows students to change variables from the command line for the following:
+- P, I, and D tuning variables
```
Comment (ctessler) [inline README.md:67]: Please provide the command line arguments with an example so that it's consistent across submissions.

File: README.md [hunk 2]
```diff
@@ -59,17 +59,48 @@ So, in summary, here's what we need to do:
...
+-  Continual terminal output delays node processing and may impact the correct operation of the vehicle.
+Provide terminal output for the successful launching of the node and no more.
+
+### Clamping
+Clamping is used to control the calculated control output, *u(t)*, which is to be considered the steering_angle.
+While testing, it is possible this number will quickly get out of hand due to integral windup. Using a clamp
+will ensure *u(t)* is always within an acceptable range.
+The most the steering_angle can turn is by 20 degrees, in both directions. A negative steering_angle
+will turn your vehicle right, and a positive angle to the left. The steering_angle is stored under the
+AckermannDrive in radians. Be sure to keep this in mind while progressing through the lab.
```
Comment (ctessler) [inline README.md:100]: I'm not sure if you want to mention this, but the integral windup can also be addressed by limiting the window across which the integral component applies. e.g. only for the past 20 publications.

**Review by ctessler — APPROVED** [review]
The comments I added are pieces that should be added in another pull request. Please do these when you get back.

File: README.md (hunk 2 above)
Comment (Breanna-Geller) [reply README.md:100]: I'd rather allow students to figure that out on their own. I don't think it's necessary for this lab to mention this in particular. Good recommendation though!

**Pull request merged**